mediator/med_map_mod.F90

```diff
@@ -386,6 +425,33 @@ subroutine med_map_routehandles_initfrom_field(n1, n2, fldsrc, flddst, mapindex,
        return
     end if
 
+    ! Output destination status field to file if requested
+    if (dststatus_print) then
```

Comment:
Is this controlled by ESMF attributes such as using nems.configure? If not, it would be nice to add it.

Reply:
I was copying what had been done for rhprint, since this is probably something you would do as a debugging step. But I can implement as a configurable attribute.